app/mailers/purchase_notifier.rb

```diff
@@ -0,0 +1,54 @@
+class PurchaseNotifier < ActionMailer::Base
+
+  include DateHelper
+  add_template_helper ApplicationHelper
+  add_template_helper TranslationHelper
+  add_template_helper OrdersHelper
+  add_template_helper ViewHookHelper
+
+  default from: Settings.email.from, content_type: "multipart/alternative"
+
+  # Notifies the specified recipient if an order is placed including a product
```

Comment:
Can we replace recipient with "facility staff member" or something similar, just to be super clear?